ang/civicase/BulkActionsMessage.js

```diff
@@ -0,0 +1,18 @@
+(function (angular, $, _) {
+  var module = angular.module('civicase');
+
+  module.directive('civicaseBulkActionsMessage', function () {
+    return {
+      restrict: 'EA',
+      replace: true,
+      controller: 'civicaseBulkActionsController',
```

Comment:
Why dont you just use `civicaseBulkActionsController` as `ng-controller` in the HTML and remove the JS file. Would that work?

Reply:
This can't be done for following reasons
* This is a directive and if we want to make the HTML reusable then its better to use it as a directive. if we use the above approach then we will have to re write the whole HTML.
* Also the attributes that are used in this directive are specific to itself and make it reusable. For e.g. `selectedItems`  for cases will have different variable name (selected-items="selectedCases.length") and for activities might be different (selected-items="selectedActivities.length"

Reply:
Lets discuss this.